endpointmanager/pkg/chplquerier/vendorquerier.go

```diff
@@ -52,8 +52,9 @@ type chplVendor struct {
 func GetCHPLVendors(ctx context.Context, store *postgresql.Store, cli *http.Client) error {
 	log.Debug("requesting vendors from CHPL")
 	prodJSON, err := getVendorJSON(ctx, cli)
+	// None of the returned errors should break the system, so just return nil
```

Comment:
More of a question than a comment - why does GetCHPLVendors not return an error while GetCHPLProducts does?

Reply:
the honest answer is that i thought i had updated them both for the CHPL requests ticket to not return errors and i apparently had not, so i'll change them to make them consistent!

Reply:
I noticed that when I took over the adjust chpl request ticket but figured that you didn't finish the changes yet. Makes sense now!

Reply:
yep exactly! it should now be updated!